### PR TITLE
Add lint

### PR DESCRIPTION
**Pull request by Minglee01**
@incuna/js please review

Added linting.  next is minification then tests

### CODE REVIEW

**Comment by wytrych** [comment]
Should we move to `eslint` maybe?

**Comment by Minglee01** [comment]
@grahamgilchrist ?

**Comment by grahamgilchrist** [comment]
> Should we move to eslint maybe?

It would be nice, since eslint is the futuree, but we'd need to translate the `.jscs` and `.jshint` rules/files to the new eslint ones.

**Comment by grahamgilchrist** [comment]
@Minglee01 Do you have time in this PR to look into that?

**Comment by grahamgilchrist** [comment]
Perhaps we could merge this one, as we know these setting work, and then another PR to update to eslint?

**Comment by Minglee01** [comment]
i dont mind.

**Comment by Minglee01** [comment]
can do i can update to eslint

**Comment by grahamgilchrist** [comment]
@Minglee01 Can you also add a `.travis.yml` to this repo to run the grunt tests? I've enabled travis for this repo.

**Comment by grahamgilchrist** [comment]
@Minglee01 Is this ready to merge?

**Comment by Minglee01** [comment]
Yes

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
+* Add file linting.
```
Comment (henrahmagix) [inline CHANGELOG.md]: This sounds like a dev ops change, maybe `Lint source to <standard>` instead?

**Comment by henrahmagix** [comment]
@Minglee01 I thought you were going to use eslint? Anything to stop us having both `jshint` and `jscs` and having to disable/enable both inline.

**Comment by Minglee01** [comment]
I am itd in a different branch

File: CHANGELOG.md (hunk 1 above)
Comment (Minglee01) [inline CHANGELOG.md]: I have a eslint branch where ive changed thia

**Pull request merged**